samples/SwiftDemoApp/SwiftDemoApp/HeatmapViewController.swift

```diff
@@ -0,0 +1,100 @@
+/* Copyright (c) 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import GoogleMaps
+import UIKit
+
+class HeatmapViewController: UIViewController, GMSMapViewDelegate {
+  private var mapView: GMSMapView!
+  private var heatmapLayer: GMUHeatmapTileLayer!
+  private var button: UIButton!
+
+  private var cameraLatitude = -37.848
```

Comment:
I still cannot see where these variables are used. The camera variable in loadView() uses the constant lat, lng in its constructor.